.github/workflows/deploy.yml

```diff
@@ -21,11 +20,26 @@ jobs:
         with:
           java-version: 1.8
 
+      - name: Configure publish keys
+        env:
+          GRADLE_PUBLISH_KEY: ${{ secrets.GRADLE_PUBLISH_KEY }}
+          GRADLE_PUBLISH_SECRET: ${{ secrets.GRADLE_PUBLISH_SECRET }}
+        shell: bash
+        run: |
+          mkdir -p ~/.gradle/
+          echo gradle.publish.key=${GRADLE_PUBLISH_KEY} >> ~/.gradle/gradle.properties
+          echo gradle.publish.secret=${GRADLE_PUBLISH_SECRET} >> ~/.gradle/gradle.properties
+
       - name: Publish
         uses: burrunan/gradle-cache-action@v1.6
         with:
           remote-build-cache-proxy-enabled: false
-          arguments: clean build bintrayUpload -s --scan
+          arguments: publishPlugins -s --scan
         env:
-          BINTRAY_API_KEY: ${{ secrets.BINTRAY_API_KEY }}
-          BINTRAY_USERNAME: ${{ secrets.BINTRAY_USERNAME }}
+          GRADLE_PUBLISH_KEY: ${{ secrets.GRADLE_PUBLISH_KEY }}
```

Comment:
Don't think these are used. Alternatively, I think you can clean up with just `-Pgradle.publish.key=${{ secrets.GRADLE_PUBLISH_KEY}} -Pgradle.publish.secret=${{ secrets.GRADLE_PUBLISH_SECRET }}` without the file mangling? GitHub scrubs secrets from all output automatically in case that's the concern.

Reply:
oh, thanks! I used the [properties block](https://github.com/burrunan/gradle-cache-action#usage) on the action instead, as that seems a little cleaner. wdyt?

Reply:
Ah yeah looks great